handlers/update_resource.go

```diff
@@ -53,6 +70,10 @@ func (d *updateResourceEntry) Handle(params operations.UpdateResourceParams) mid
 	return operations.NewUpdateResourceOK().WithPayload(response)
 }
 
+func (d *updateResourceEntry) persistResource(r *datautils.Resource) error {
```

Comment:
Is it worth breaking this out into a function call?  Seems like too much indirection, since this just calls `d.database.Insert()`